distribution/test/itests/test-itests-common/src/main/resources/oidcMetadata.json

```diff
@@ -0,0 +1,112 @@
+{
+  "issuer": "http://localhost:{{IDP_PORT}}/auth/realms/master",
+  "authorization_endpoint": "http://localhost:{{IDP_PORT}}/auth/realms/master/protocol/openid-connect/auth",
+  "token_endpoint": "http://localhost:{{IDP_PORT}}/auth/realms/master/protocol/openid-connect/token",
+  "token_introspection_endpoint": "http://localhost:{{IDP_PORT}}/auth/realms/master/protocol/openid-connect/token/introspect",
+  "userinfo_endpoint": "http://localhost:{{IDP_PORT}}/auth/realms/master/protocol/openid-connect/userinfo",
+  "end_session_endpoint": "http://localhost:{{IDP_PORT}}/auth/realms/master/protocol/openid-connect/logout",
+  "jwks_uri": "http://localhost:{{IDP_PORT}}/auth/realms/master/protocol/openid-connect/certs",
+  "check_session_iframe": "http://localhost:{{IDP_PORT}}/auth/realms/master/protocol/openid-connect/login-status-iframe.html",
+  "grant_types_supported": [
+    "authorization_code",
+    "implicit",
+    "refresh_token",
+    "password",
+    "client_credentials"
+  ],
```

Comment:
📘 In one of our future meetings, we should verify that we actually support all these grant types, or if there are any more grant types we need to support.